M/MPICH/build_tarballs.jl

```diff
@@ -1,32 +1,26 @@
 using BinaryBuilder, Pkg
 
 name = "MPICH"
-version = v"3.3.2"
+version = v"3.4.2"
 
 sources = [
     ArchiveSource("https://www.mpich.org/static/downloads/$(version)/mpich-$(version).tar.gz",
-                  "4bfaf8837a54771d3e4922c84071ef80ffebddbb6971a006038d91ee7ef959b9"),
-    DirectorySource("./bundled"),
+                  "5c19bea8b84e8d74cca5f047e82b147ff3fba096144270e3911ad623d6c587bf"),
 ]
 
 script = raw"""
 # Enter the funzone
-cd ${WORKSPACE}/srcdir/mpich-*
+cd ${WORKSPACE}/srcdir/mpich*
 
-if [[ "${target}" == powerpc64le-* ]]; then
+#if [[ "${target}" == powerpc64le-* ]]; then
     # I don't understand why, but the extra link flags we append in the gfortran
     # wrapper confuse the build system: the rule to build libmpifort has an
     # extra lone `-l` flag, without any library to link to.  The following sed
     # script basically reverts
     # https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/749, so that the
     # extra link flags are not appended to the gfortran wrapper
-    sed -i 's/POST_FLAGS+.*/POST_FLAGS=()/g' /opt/bin/gfortran
```

Comment:
Why did you remove this?  The comments suggests this is the cause of the current issue with PowerPC

Reply:
See #1022

Reply:
That was causing all the powerpcs to fail. Now one of them passes. :shrug:

Reply:
This is what happens if I bring that line back. Perhaps the path of /opt/bin/gfortran has changed?

https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=11758&view=logs&j=16771ab9-fdae-5fdd-eb0c-d2e6527e22cf&t=da4cd3ba-1dac-5953-6278-2bd0a86ec74c&l=15521

Reply:
It's failing because we moved the file